.github/workflows/ci.yml

```diff
@@ -94,6 +94,10 @@ jobs:
       matrix:
         arch: [x64]
     steps:
+    - name: Setup dotnet
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: '5.0.x'
     - uses: ilammy/msvc-dev-cmd@v1
```

Comment:
not your fault, but

`ilammy/msvc-dev-cmd` <-- in my experience, this github action has no effect

it ended up in all our ci.yml files when we were first setting up our repos and copy/pasting everything

later we discovered it had no effect, but we've never made a concerted effort to fix this across our 20+ repos :(

Reply:
https://tenor.com/en-GB/view/coding-programming-pink-panther-when-you-delete-a-block-of-code-that-you-though-was-useless-gif-17338075